pkg/crd/crd.go

```diff
@@ -244,13 +244,17 @@ func (c *Client) Start(exit <-chan struct{}) {
 	go c.BindingInformer.Run(exit)
 	go c.IDInformer.Run(exit)
 	go c.AssignedIDInformer.Run(exit)
-	c.SyncCache(exit)
+	c.SyncCache(exit, true)
 	glog.Info("CRD informers started")
 }
 
 // SyncCache synchronizes cache
-func (c *Client) SyncCache(exit <-chan struct{}) {
+func (c *Client) SyncCache(exit <-chan struct{}, initial bool) {
 	if !cache.WaitForCacheSync(exit) {
+		if !initial {
+			glog.Error("Cache could not be synchronized")
```

Comment:
StartLite can get called from NMI, which will result in glog call instead of logrus call. We could use the log interface (like in the probe) to use the right logging calls.